data_models/run_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# script for executing Django Unittests within Docker container.
+
+docker build -t models_tests -f Dockerfile.tests .
+
+HOST_IP=$(ifconfig eth0 | grep "inet " | awk -F'[: ]+' '{ print $4 }')
+docker run \
+       --add-host=database:$HOST_IP \
+       --env-file /home/kurt/Development/data_refinery/data_models/environments/test \
+       -i models_tests "$@"
```

Comment:
Good catch, I fixed this for [`run_shell`](https://github.com/data-refinery/data_refinery/pull/6/commits/6c1e3cb11b3cb484306fd926cfe533d96f170261) but forgot I had done something similar here.